### ISSUE TITLE
How to use this repo

### ISSUE THREAD

**Issue by SMABHISHEK** [goldrooster/IOTA-Empirical-Data-Analysis#1]
Requesting you to help me understand how to recreate the IOTA networkX graph.

Please add step by step instructions to create IOTA networkX graph.

**Comment by goldrooster** [comment]
Hi, you should first extract the hash value of transactions from the raw datasets, then create the Tangle based on the hash. For example, each transaction includes a branch hash and a trunk hash, and itself hash. I have labeled each transaction hash with a number. For example, transaction A is 1, transaction B is 2, and transaction C is 3, in transaction A, the branch and trunk are B and C, then I write down the (1,2) and (1,3) as edges in Networkx and build the tangle. Which step are you confused about?

**Comment by SMABHISHEK** [comment]
Hello Goldrooster,

Thanks for your response. 

I am trying to figure out what are the series of python files to go with ?
https://github.com/goldrooster/IOTA-Empirical-Data-Analysis/tree/master/real_tangle/calculate_data\

Should it be like -
1. Download the Tangle data from http://dertangle.iota.cafe/ Example: http://dertangle.iota.cafe/0000000-0006000.tgz
2.  Use Extract_Data_from_Raw_Dataset_940227.py ?

I request you to please help me with how to proceed. What are the series of actions and python files to go with, till it can be NetworkX graph

**Comment by goldrooster** [comment]
Yes, you can first download the raw dataset file, and use the Extract_Data_from_Raw_Dataset_940227.py to extract the raw data to a .json file, then use the 00MS_branch_trunk.py  to get the branch edges and trunk edges, after that, you can use 04MS_indegree_dict.py to re-build the tangle.
